lib/workflow-executor.ts

```
@@ -640,15 +640,31 @@ export class WorkflowExecutor {
           }
         }
 
+        // Get the model reference for SQLite queries
+        const modelForQuery = inputValues.input && !Array.isArray(inputValues.input) && inputValues.input.id
+          ? inputValues.input
+          : undefined;
+
+        console.log("🔧 About to call manageProperties with:", {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)